.travis.yml

```diff
@@ -27,6 +27,7 @@ env:
 matrix:
   include:
     - env: RUNTIME=3.6 TOOLKITS="pyqt pyqt5 pyside2 wx"
+    - env: RUNTIME=3.6 TOOLKITS="pyqt5" TRAITS_REQUIRES="^=6.0"
```

Comment:
given that we're not skipping wx tests, does it make sense to run wx too with traits 6.0.x?

Reply:
Yep.